src/services/earn/types.ts

```diff
@@ -62,3 +63,59 @@ export type AddFundsEarn = { swapConfig?: EarnActionSwapConfig } & (
   | { permitData: PermitData['permitData']; signature: string }
   | { token: TokenAddress; amount: BigIntish }
 );
+
+export type Strategy = {
+  id: StrategyId;
+  chainId: ChainId;
+  depositTokens: TokenAddress[];
```

Comment:
These should probably be of type `Token` (we should make sure they are returned as part of `tokens` in the API payload)